packages/opentelemetry-sdk-node/src/index.ts

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { metrics } from '@opentelemetry/api';
+import { MeterProvider } from '@opentelemetry/metrics';
+import { NodeTracerProvider } from '@opentelemetry/node';
+import { detectResources, Resource } from '@opentelemetry/resources';
+import { BatchSpanProcessor } from '@opentelemetry/tracing';
+import { SDKConfiguration } from './types';
+
+export * as api from '@opentelemetry/api';
+export * as contextBase from '@opentelemetry/context-base';
+export * as core from '@opentelemetry/core';
+export * as metrics from '@opentelemetry/metrics';
+export * as node from '@opentelemetry/node';
+export * as resources from '@opentelemetry/resources';
+export * as tracing from '@opentelemetry/tracing';
+
+export async function configure(
```

Comment:
Should we rename this so that it is clear that this is the only function that needs to be called, e.g. `start`? WDTY?

Reply:
Think I could go either way on this since you are technically "configuring" the sdk also. What about this?

```
const sdk = opentelemetry.configure({
  opt1: true,
});

// having this as a separate function would allow users to avoid async if they want to
await sdk.autoDetectResources()

// this actually calls the API methods to set the providers, propagator, etc
sdk.start();
```